[2025-09-25T20:44:11.528+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-25T20:44:11.534+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hw1.collect_data manual__2025-09-25T20:44:09.043213+00:00 [queued]>
[2025-09-25T20:44:11.538+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hw1.collect_data manual__2025-09-25T20:44:09.043213+00:00 [queued]>
[2025-09-25T20:44:11.538+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-25T20:44:11.543+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): collect_data> on 2025-09-25 20:44:09.043213+00:00
[2025-09-25T20:44:11.547+0000] {standard_task_runner.py:64} INFO - Started process 74924 to run task
[2025-09-25T20:44:11.548+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hw1', 'collect_data', 'manual__2025-09-25T20:44:09.043213+00:00', '--job-id', '199', '--raw', '--subdir', 'DAGS_FOLDER/kudryakov_hw1.py', '--cfg-path', '/tmp/tmpo08a_ji9']
[2025-09-25T20:44:11.549+0000] {standard_task_runner.py:91} INFO - Job 199: Subtask collect_data
[2025-09-25T20:44:11.574+0000] {task_command.py:426} INFO - Running <TaskInstance: hw1.collect_data manual__2025-09-25T20:44:09.043213+00:00 [running]> on host 2b301b74205f
[2025-09-25T20:44:11.611+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='VLADISLAV KUDRYAKOV' AIRFLOW_CTX_DAG_ID='hw1' AIRFLOW_CTX_TASK_ID='collect_data' AIRFLOW_CTX_EXECUTION_DATE='2025-09-25T20:44:09.043213+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-25T20:44:09.043213+00:00'
[2025-09-25T20:44:11.611+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-25T20:44:11.620+0000] {kudryakov_hw1.py:78} INFO - Началась загрузка данных
[2025-09-25T20:44:11.646+0000] {kudryakov_hw1.py:85} INFO - Данные получили
[2025-09-25T20:44:11.653+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2025-09-25T20:44:11.654+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2025-09-25T20:44:12.880+0000] {kudryakov_hw1.py:94} INFO - Данные успешно загружены
[2025-09-25T20:44:12.883+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-25T20:44:12.883+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-25T20:44:12.889+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=hw1, task_id=collect_data, run_id=manual__2025-09-25T20:44:09.043213+00:00, execution_date=20250925T204409, start_date=20250925T204411, end_date=20250925T204412
[2025-09-25T20:44:12.906+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-25T20:44:12.921+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-25T20:44:12.921+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
